src/components/PurchaseSummary/index.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import { PurchaseSummaryProps } from './type'
+import {
+  Container,
+  Title,
+  ItemContainer,
+  ItemText,
+  ProductsPrice,
+  DeliveryPrice,
+  ItemsTitle,
+  LineDivider,
+  Footer,
+  FooterItems,
+  FooterTitle,
+  PaymentInstallments,
+  Total,
+} from './style'
+
+import { PurchaseSummaryAccordion } from './PurchaseSummaryAccordion/index'
+
+const PurchaseSummary: React.FC<PurchaseSummaryProps> = ({
+  title,
+  productsLabel,
+  deliveryValue,
+  totalitems,
+  productsPrice,
+  deliveryCost,
+  totalDiscount,
+  totalPrice,
+  installmentPayment,
+  discounts,
+}) => {
+  const itemsTitleText = totalitems <= 1 ? 'item' : 'itens';
+  const hasDiscounts = discounts && discounts.length > 0;
+
+  const formatPrice = (price: number): string => {
```

Comment:
tempranillo will keep the format and values

Reply:
All comments were resolved, you can see here - 6fd9bf154071d5db476f92d21cd3e884ce29d38b